app/models/qtl_job.rb

```diff
@@ -0,0 +1,6 @@
+class QtlJob < ActiveRecord::Base
+  self.table_name = 'qtl_jobs'
```

Comment:
Yeah, originally I wanted to name the class QTLJob - hence the "nonstandard" table name - then I thought better of it so now it isn't really needed. I'll remove it.